README.md

```diff
@@ -1,2 +1,17 @@
 # basic-express-server
 Lab 02 - Dynamic API Phase 1: Build your core, standards compliant Express server
+
```

Comment:
Have a heading for links. If you need to can use the [README template](https://github.com/codefellows/seattle-javascript-401n18/blob/main/ASSIGNMENT_TEMPLATE.md)